### PR TITLE
Improve digest discovery portion of docs

### PR DESCRIPTION
**Pull request by tom-haines**
Improvement to documentation regarding digest discovery during [#2751 discussion](https://github.com/GoogleContainerTools/jib/issues/2751).

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -557,6 +557,35 @@ $ docker manifest inspect openjdk:8
...
+         ...
+         "digest": "sha256:1fbd49e3fc5e53154fa93cad15f211112d899a6b0c5dc1e8661d6eb6c18b30a6",
+         "platform": {
+            "architecture": "arm64",
+            "os": "linux",
+            "variant": "v8"
+         }
+      }
+   ]
+}
```
Comment (mpeddada1) [inline docs/faq.md:585]: Perhaps we can remove this section since we've already mentioned this above.

**Review by mpeddada1 — COMMENTED** [review]
Thank you for making our docs more comprehensive!

File: docs/faq.md (hunk 1 above)
Comment (tom-haines) [reply docs/faq.md]: Good suggestion thanks @mpeddada1 .  I've updated to make relevance of output clearer  (eee6882), and removed duplication.

File: docs/faq.md [hunk 2]
```diff
@@ -557,6 +557,22 @@ $ docker manifest inspect openjdk:8
 }
 ```
 
+You can inspect the digest (`@sha256:...`) of a manifest image with the `--verbose` option.
```
Comment (chanseokoh) [inline docs/faq.md:560]: I don't think this is necessary. Without `--verbose`, you can already see the digest of each image in the list already as seen above.

**Comment by mpeddada1** [comment]
@tom-haines Apologies for making you do the edits. But after seeing @chanseokoh's comment, I agree that including the `--verbose` flag may be a little redundant since info about the digest is already present above. Closing this PR but appreciate your attempt.

**Pull request closed without merging**